pytest-shutil/setup.py

```diff
@@ -25,7 +25,7 @@
                     'execnet',
                     'contextlib2;python_version<"3"',
                     'pytest',
-                    'path.py',
+                    'path',
```

Comment:
You could use this approach to begin the migration until Python 2.7 support is dropped:

```suggestion
                    'path; python_version >= "3.5"',
                    'path.py; python_version < "3.5",
```

Reply:
Thanks @jaraco, I don't know why I didn't think of that myself! :laughing: